communication/mavlink_waypoint_handler.cpp

```diff
@@ -186,20 +164,20 @@ void Mavlink_waypoint_handler::receive_count(Mavlink_waypoint_handler* waypoint_
         if (waypoint_handler->waypoint_receiving_ == false)
```

Comment:
Is this correct?
As of the [waypoint protocol](http://qgroundcontrol.org/mavlink/waypoint_protocol), the message MISSION_COUNT initiates a transfer. And I think the ground station acts as a master, so if it decides to resume the transfer, we should re-start sending the waypoints from 0.